public/app/percona/ui-events/reducer.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-explicit-any,@typescript-eslint/consistent-type-assertions, */
+
```

Comment:
Check our reducers at `public/app/percona/shared/core/reducers`.
Let's put this one there with a similar pattern, we don't need to check prefixes and the logic should be simpler using a state slice.

Reply:
It is simple and flexible, I need access to all events. In `percona/shared/core/reducers` they are scoped.